crates/core/migrations/2022-04-27-183240_create_graph_program_columns/up.sql

```diff
@@ -0,0 +1,11 @@
+alter table graph_connections
+add column connected_at timestamp,
+add column disconnected_at timestamp;
+
+create index on graph_connections (connected_at);
```

Comment:
@kevinrodriguez-io disconnected_at is the field we are more likely to search against. In fact it will be where disconnected_at is null so doing on index on null or not would be the most efficient search path.

Can keep connected_at if we do sorting.